app/controllers/admin/sales/invoices.js

```diff
@@ -38,7 +37,12 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
       {
         name      : 'Status',
         valuePath : 'status'
-      }
+      },
+//       {
+//         name            : 'Action',
+//         valuePath       : 'identifier',
+//         cellComponent   : 'ui-table/cell/admin/sales/cell-action'
+//       }   
```

Comment:
Codacy found an issue: [Expected indentation of 6 spaces but found 0.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6173716)